### PR TITLE
Metadata templates

### PR DESCRIPTION
**Pull request by rsevilla87**
This PR includes the following changes:

- The Metadata initContainers YAML now uses a common "pretty" template included from each benchmark. This will simplify the code and improve maintenance. I had to convert a couple of templates from pods to jobs due to YAML linting issues.
- Add stockpileSkipTags and stockpileTags parameter
- Update CRD's metadata section.
- Remove unused valid_uperf_store.yaml test file
- Configure stockpileTags with ["common", "k8s", "openshift"]` by default
- Update metadata documentation

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Comment by rsevilla87** [comment]
@dry923 @jtaleric  PTAL

File: docs/metadata.md [hunk 1]
```diff
@@ -9,32 +9,51 @@ How To:
...
 
 The metadata collection is done through the use of [Stockpile](https://github.com/cloud-bulldozer/stockpile), [Backpack](https://github.com/cloud-bulldozer/backpack) and [Scribe](https://github.com/cloud-bulldozer/scribe).
 The data is then uploaded to a defined Elasticsearch instance.
 
+Metadata collection is enabled by default, it can be disabled setting collection to false in the metadata section:
+
+```yaml
+metadata:
+  collection: false
+```
```
Comment (dry923) [inline docs/metadata.md:22]: I'm good with enabling metadata collection by default however would like to get @jtaleric , @bengland2 , @chaitanyaenr and @mohit-sheth 's thoughts as well

File: resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml [hunk 2]
```diff
@@ -40,15 +40,47 @@ spec:
...
+                  serviceaccount:
+                    default: default
+                    type: string
+                  stockpileSkipTags:
+                    default: []
+                    items:
+                      type: string
+                    type: array
+                  stockpileTags:
+                    default: ["memory", "k8s", "openshift"]
```
Comment (dry923) [inline resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml:69]: why not "common" in the default tags as well?

Comment (rsevilla87) [reply resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml:69]: Sure, had a bug here :). The documentation said `["common", "k8s", "openshift"]`.

**Pull request merged**